packages/react-components/src/components/ImageOverlay.tsx

```diff
@@ -106,7 +111,7 @@ export const ImageOverlay = (props: ImageOverlayProps): JSX.Element => {
 
   const renderHeaderBar = (): JSX.Element => {
     return (
-      <Stack className={mergeStyles(headerStyle)} role="heading" aria-label={title || 'Image'} aria-level={2}>
```

Comment:
Do we still need it? 
It looks like it wasused during the keyboard navigation

Reply:
you might need the role and the level but if the announcer is using the string then the label can be removed here.